projects/foto-finder-final.md

```diff
@@ -0,0 +1,205 @@
+* [ ]  Advanced Beginner - Application implements all major comp details accurately and correctly on desktop and mobile (colors, fonts, icons,spacing, alignment,  etc.). Transitions between screen sizes may not be smooth.
+* [ ]  Proficient - Application implements all major comp details accurately and correctly on desktop and mobile (colors, fonts, icons,spacing, alignment,  etc.) with smooth transitions between screen sizes. Additional elements added generally match the visuals established in the comps, but may be slightly awkward.
+* [ ]  Exceptional - Application implements all major comp details accurately and correctly on desktop and mobile (colors, fonts, icons,spacing, alignment,  etc.) with smooth transitions between screen sizes. Additional elements that have been added match the visuals established in the comps.
+
+------------------------------------------------------------------
+
+### HTML - Style and Implementation
+
+* [ ]  Novice - Crafts markup according to the [turing html style guide](https://github.com/turingschool-examples/html)
+* [ ]  Advanced Beginner - Application adds to the above by using `data-*` attributes for all data related things
+* [ ]  Proficient - Applications adds to the above with markup that is easy to read and follow across naming conventions
+* [ ]  Exceptional - Application adds to the above by using BEM, SMACCS, or another set of naming conventions for classes
+    * [ ]  Implements html that is accessible for folks with visual disabilities. Reference [this lesson plan](http://frontend.turing.io/lessons/floating/web-accessibility.html)
+
+------------------------------------------------------------------
+
+### CSS - Style and Implementation
+
+* [ ]  Novice - Crafts CSS according to the [turing css style guide](https://github.com/turingschool-examples/css)
+* [ ]  Advanced Beginner - Application adds organization for the whole stylesheet and within rules
+* [ ]  Proficient - Applications adds to the above by removing repetitive rules and blocks of code according to the DRY principle 
+* [ ]  Exceptional - Application adds to the above by using BEM, SMACCS, or another set of naming conventions for classes
+
+------------------------------------------------------------------
+
+### JAVASCRIPT - Style and Implementation
+
+* [ ]  Novice - Crafts JS according to the [turing js style guide](https://github.com/turingschool-examples/javascript/tree/master/es5)
+* [ ]  Advanced Beginner - Application correctly implements data model for the `Photo` class including all required methods
+* [ ]  Proficient - Application adds readability by incorporating both DRY and SRP practices
+  * [ ]  All functions are less than 10 lines
+  * [ ]  There are less than 3 global variables
+  * [ ]  There are no nested if/else statements
+* [ ]  Exceptional - Application refactors all for loops into the proper array prototype iteration method
+   * [ ] Using logical operators instead of if/else
```

Comment:
May need to provide more clarity here since technically comparison operators do exist in if/elses as well